### PR TITLE
docs: update document bullet formatting

### PR DESCRIPTION
**Pull request by essjay05**
Update bulleted points in CODE_REVIEWS.md to remove '.' at the end of each point to be cohesive with CONTRIBUTING.md guidelines.

### CODE REVIEW

File: CODE_REVIEWS.md [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
 ## Workflow
+
 1. The code author sends a PR for review. This request should include:
-  * A high-level description of the change being made.
-  * Links to any relevant issues.
-  * Screenshots (for visual changes or new additions)
+
+- A high-level description of the change being made
+- Links to any relevant issues
+- Screenshots (for visual changes or new additions)
```
Comment (devversion) [inline CODE_REVIEWS.md:15]: Should this be indented to make clear it's a sub-bullet point to (1)?

**Review by devversion — APPROVED** [review]

**Review by crisbeto — APPROVED** [review]

**Comment by crisbeto** [comment]
@essjay05 can you rebase the PR? I just tried merging it and got some conflicts.

**Comment by essjay05** [comment]
> @essjay05 can you rebase the PR? I just tried merging it and got some conflicts.
@crisbeto Done! Let me know if you need anything else from my end.

**Comment by essjay05** [comment]
New cleaned up PR to replace this one can be found here: https://github.com/angular/components/pull/28123

**Pull request closed without merging**